storybook/src/elements/nuxeo-data-table/nuxeo-data-table-structure.js

```diff
@@ -0,0 +1,52 @@
+import { html } from 'lit-html';
+
+export const tableTemplate = (params) =>
+  html`
+    <nuxeo-data-table
+      .items="${params.properties.data}"
+      ?editable="${params.editable}"
+      ?orderable="${params.orderable}"
+      ?settings-enabled="${params.settingsEnabled}"
+      ?selection-enabled="${params.selectionEnabled}"
+      ?multi-selection="${params.multiSelection}"
+      ?details-enabled="${params.detailsEnabled}"
+      label="${!params.label ? '' : params.label}"
+      ?required="${params.required}"
+    >
+      <nuxeo-data-table-column name="Company">
+        <template>
+          [[item.company_name]]
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-column name="Date" ?hidden="${params.hidden}">
+        <template>
+          <nuxeo-date datetime="[[item.date]]"></nuxeo-date>
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-column name="Department" order="${params.orderColumn}">
+        <template>
+          [[item.department]]
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-column name="City" ?align-right="${params.alignRight}">
+        <template>
+          [[item.city]]
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-column name="User" flex="${params.flex}">
+        <template>
+          [[item.user]]
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-form>
+        <template>
+          <nuxeo-input value="{{item}}" label="Multi String" type="text"></nuxeo-input>
```

Comment:
This won't do anything good, since were binding a `nuxeo-input` to item, which is an object with a user, a city, a department, a date and a company name. Here we should have an input for each of those fields:
```suggestion
          <nuxeo-input value="{{item.user}}" label="User" type="text"></nuxeo-input>
          ...
```